src/navigations/root-stack.js

```diff
@@ -246,7 +246,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="TopicPageScreen"
-          component={withKeyboardWrapper(TopicPageScreen)}
+          component={TopicPageScreen}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `withKeyboardWrapper` from the `TopicPageScreen` component might cause issues if this HOC was handling keyboard events for the screen. Please ensure that the keyboard events are handled properly in the `TopicPageScreen` itself or another wrapper is taking care of it.
```diff
- component={withKeyboardWrapper(TopicPageScreen)}
+ component={TopicPageScreen}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->